doc/Manuals/ig-csha_x-road_6_ha_installation_guide.md

```diff
@@ -398,10 +399,9 @@ See [Central Server User Guide](ug-cs_x-road_6_central_server_user_guide.md#17-m
 
 Edit `/etc/xroad/db.properties` and change the connection properties:
 ```properties
-username=centerui
-password=<password>
-database=centerui_production
-host=<master host>
+spring.datasource.username=centerui
+spring.datasource.password=<password>
+spring.datasource.url=jdbc:postgresql://<master host>:<master port>/centerui_production
```

Comment:
Should we use the `<database>` placeholder instead of `centerui_production` since the database name is configurable now?

```
spring.datasource.url=jdbc:postgresql://<master_host>:<master_port>/<database>
```